.github/workflows/test.yml

```diff
@@ -32,14 +32,14 @@ jobs:
 
       - name: "Run tests for ${{ matrix.modsec_version }}"
         run: |
-          mkdir -p tests/logs/${{ matrix.modsec_version }}/{nginx,apache2}
+          mkdir -p tests/logs/${{ matrix.modsec_version }}
```

Comment:
These directories can be necessary while the web server launches, before the configuration is loaded. They are the default log directories. Web server startup may fail without them.

Reply:
Added back in [2e262ec](https://github.com/coreruleset/coreruleset/pull/3251/commits/2e262ecd4279a6efe35ea757e8e9440c2b49ce4c)